src/test/validation_tests.cpp

```diff
@@ -4,7 +4,6 @@
 
 #include <chainparams.h>
 #include <consensus/amount.h>
-#include <net.h>
```

Comment:
was this header not needed? I don't see `#include <node/connection.h>` being added

Reply:
Seems like it wasn't; iwyu doesn't recommend adding it back, either. Shouldn't be needed -- validation stuff should be "kernel" level, so shouldn't rely on any p2p/net things.

Reply:
sounds good to me!